lightstep/resource_inferred_service_rule.go

```diff
@@ -54,7 +53,7 @@ func getResourceInferredServiceRuleSchema() map[string]*schema.Schema {
 			MinItems: 1,
 		},
 		"group_by_keys": {
-			Description: "A list of attribute keys whose values will be included in the inferred service name",
+			Description: "Attribute keys whose values will be included in the inferred service name",
```

Comment:
tfplugindocs automatically adds the types to the doc, so no need to write that this is a list